packages/core/src/bodl/bodl-emitter-service.ts

```diff
@@ -120,6 +120,10 @@ export default class BodlEmitterService implements BodlService {
         this.bodlEvents.emit('bodl_checkout_email_entry_began');
     }
 
+    customerSuggestionInit(payload?: BodlEventsPayload) {
+        this.bodlEvents.emit('bodl_checkout_customer_suggestion_initialization', payload);
```

Comment:
I think we have an opportunity as a follow up to have proper typing for these events as well. Maybe we can tackle it in a future iteration?